Plots/Streamlines/NCL_stream_9.py

```diff
@@ -0,0 +1,80 @@
+"""
+NCL_stream_9.py
+===============
+This script illustrates the following concepts:
+   - Showing features of the new color display model
+   - Using opacity to emphasize or subdue overlain features
+   - Using stLevelPalette resource to assign a color palette
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/stream_9.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/stream_9_1_lg.png
+
+"""
+
+################################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import matplotlib.cm as cm
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+import matplotlib.colors as colors
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+################################################################################
+# Make color map 
+
+colormap = colors.ListedColormap(['darkblue', 'mediumblue', 'blue', 'cornflowerblue', 'skyblue', 'aquamarine',
+        'lime', 'greenyellow', 'gold', 'orange', 'orangered', 'red', 'maroon'])
+
+colorbounds = [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 52]
```

Comment:
Is it possible to do np.arange with these numbers? It may solve the color bar labels using 0 and 52 (number wise) but not being that way on the original projection. If not, I don't think it would be a problem to leave it as is.